pipelines/data_embedding_pipeline/config_data_embedding_pipeline.yaml

```diff
@@ -2,6 +2,6 @@ steps:
   load_data:
     enable_cache: False
     parameters:
-      data_version: "data/first_version"
-      reference_data_version: "data/first_version"
+      data_version: "data/second_version"
```

Comment:
Why the change of version?

Reply:
We have performed data cleaning so we are creating a new tagged dvc version for our experiment.